ui/src/appV2/components/ManifestsTableTable/ManifestsTable.tsx

```diff
@@ -2,208 +2,128 @@ import { useManifests } from '@appV2/components/ManifestsTableTable/useSboms';
+        { text: '50', value: 50 },
+        { text: '100', value: 100 },
+      ]}
+      totalItems={total || 0}
+      onChange={({ page, pageSize: newPageSize }) => {
+        if (page !== pageIndex) {
+          onSetPage(page);
+        } else if (newPageSize !== pageSize) {
+          onPerPageSelect(newPageSize);
+        }
+      }}
+    />
   );
 
-  const select = <Select
-    id="single-select"
-    isOpen={selectIsOpen}
-    selected={selectValue as ManifestsQueryType}
-    onSelect={onSelect}
-    onOpenChange={(isOpen) => setSelectIsOpen(isOpen)}
-    toggle={toggle}
-    shouldFocusToggleOnSelect
-  >
-    <SelectList>
-      <SelectGroup>
-        <SelectOption value={ManifestsQueryType.NoFilter}>{ManifestsQueryType.NoFilter}</SelectOption>
-      </SelectGroup>
-      <SelectGroup>
-        <SelectOption value={ManifestsQueryType.Purl}>{ManifestsQueryType.Purl}</SelectOption>
-      </SelectGroup>
-    </SelectList>
-  </Select>
-
-
-  const onChange = (value: string) => {
-    setSearchBarValue(value)
-  };
 
-  const searchBar = <SearchInput
-    placeholder="Enter selected id"
-    value={searchBarValue}
-    onChange={(_event, value) => onChange(value)}
-    onClear={() => onChange('')}
-    isAdvancedSearchOpen={!searchBarVisible}
-    onKeyDown={(event: React.KeyboardEvent) => {
-      if (event.key == 'Enter') {
-        onSearchCall();
-      }
-    }
-    }
-  />
-
-  const searchButton = <Button
-    variant='primary'
-    onClick={() => onSearchCall()}>Search</Button>
-
-  const pagination = <Pagination
-      itemCount={total}
-      widgetId="manifests-table-pagination"
-      perPage={pageSize}
-      page={pageIndex}
-      variant={PaginationVariant.bottom}
-      onSetPage={onSetPage}
-      onPerPageSelect={onPerPageSelect}
+  const table = (
+    <DataTable
+      rows={value || []}
+      headers={[
+        { key: 'id', header: columnNames.id },
+        { key: 'creationTime', header: columnNames.creationTime },
+      ]}
+      render={({ rows, headers }) => (
+        <TableContainer title="Latest manifests">
+          <Table aria-label="Manifests Table">
+            <TableHead>
+              <TableRow>
+                {headers.map(header => (
+                  <TableHeader key={header.key}>{header.header}</TableHeader>
+                ))}
+              </TableRow>
+            </TableHead>
+            <TableBody>
+              {value && value.map((manifest) => {
+                return (
+                  <TableRow key={manifest.id}>
+                    <TableCell>
+                      <Link to={`/manifests/${manifest.id}`}>
+                        <pre>{manifest.id}</pre>
+                      </Link>
+                    </TableCell>
+                    <TableCell>
+                      <span>
+                        {manifest?.created
+                          ? timestampToHumanReadable(Date.now() - new Date(manifest.created).getTime(), false, 'ago')
```

Comment:
I think this is less relevant when events and manifests are happening more frequently but I think the granularity of `timestampToHumanReadable` should be variable. 

`48 days 17 hours 52 minutes ago` is too much information as I'd find the time stamp much more useful, Id suggest 

* If its more than 3 days then only have the days
* If its less than 3 days but more than 1 day have day and hour
* If its less than 1 day have hours and minutes

I think an alternative is to use the ISO datetime string of the creation or trigger date and have the timestamp as a tool tip as its unlikely anyone will need to C&P the human readable delta but its still very useful information
*

Reply:
I like this idea!